packages/big-design/src/components/FileUploader/DropZone.tsx

```diff
@@ -149,6 +151,33 @@ export const DropZone = ({
     );
   }, [description]);
 
+  const renderedAction = useMemo(() => {
+    if (!action) {
+      return null;
+    }
+
+    const { label, onClick, ...actionProps } = action;
+
+    return (
+      <StyledButton
```

Comment:
Let's make sure to put `type="button"` on each of these `StyledButton`'s so it doesn't submit a form.

Reply:
Done